client/components/Questions.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+export default React.createClass({
+  propTypes: {
+    questions: React.PropTypes.array
```

Comment:
Doesn't `PropTypes` provide the way to define the type of object contained in this array?